subsys/fs/Kconfig

```diff
@@ -12,6 +12,20 @@ config FILE_SYSTEM
 	help
 	  Enables support for file system.
 
+config SYS_LOG_FS_LEVEL
+	int
+	prompt "File System log level"
+	depends on SYS_LOG
+	default 0
+	help
```

Comment:
How about adding a range check "range 0 4" here?

Reply:
I just reused existing stuff, may we can do this at kernel level as one PR.